spec/base_spec.rb

```diff
@@ -0,0 +1,80 @@
+require_relative '../lib/base'
+
+RSpec.describe Ipfs::Base16 do
+  #let(:invalid_byte_set) { 'fezhgzhze6351498' }
+  #let(:valid_byte_set) { 'cbdafe651498465bcdeaf' }
+  #let(:invalid_byte_set) { 'uluiil1658gfh' }
+  #let(:valid_byte_set) { 'f5c1d23ea6e9' }
+  let(:invalid_byte_set) { '1968loifefdsli515688' }
+  let(:valid_byte_set) { 'b65e879f65d98a15c5f6d0e9b4a5d' }
+
+  it 'has the hexadecimal alphabet' do
+    expect(described_class::ALPHABET).to eq '0123456789abcdef'
+  end
+
+  #describe '.decode' do
+  #  context 'when the bytes are an empty string' do
+  #    it 'responds with an empty array' do
+  #      expect(described_class.decode('')).to eq []
+  #    end
+  #  end
+  #
+  #  context 'when the byte is a single character string equal to `f`' do
+  #    it 'responds [15]' do
+  #      expect(described_class.decode 'f').to eq [15]
+  #    end
+  #  end
+  #
+  #  context 'when the byte is a two characters string equal to `fe`' do
+  #    it 'responds [254]' do
+  #      expect(described_class.decode 'fe').to eq [254]
+  #    end
+  #  end
+  #
+  #  context 'when the byte is a two characters string equal to `fed`' do
+  #    it 'responds [254, 13]' do
+  #      expect(described_class.decode 'fed').to eq [254, 13]
+  #    end
+  #  end
+  #end
+  #
+  # describe '.encode' do
+  #   it 'respond when the collection is empty' do
+  #     expect(described_class.encode([])).to eq ''
+  #   end
+  #
+  #   context 'takes a byte equal to 18' do
+  #     it 'responds with 0x12' do
+  #       expect(described_class.encode([18])).to eq '12'
+  #     end
+  #   end
+  # end
```

Comment:
comments in commit

Reply:
Sorry about that, forgot to take them off